### PR TITLE
NewsFeed form and Description

### PR DESCRIPTION
**Pull request by saphal1998**
Screenshot 2020-06-04 at 1 03 35 AM

#14 

The Fields need to be incorporated in the Projects Component. 

TODO: 
- [ ] Connect the projects to DB for persisting storage

### CODE REVIEW

File: newsfeed/src/components/DetailsForm.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	}
+
+	onFinish(values) {
+		const project = {
+			name: values.name,
+			repository: values.repository,
+			github: values.github,
+			tags: values.tags.split(',').map((tag) => tag.trim()),
+			date: values.date,
+		};
```
Comment (saphal1998) [inline newsfeed/src/components/DetailsForm.js:25]: @kunal-kushwaha Notice the project variable, that is where the new project is stored, so if we make a DB call here, we can save the new projects

**Comment by anikethsaha** [comment]
@saphal1998 once https://github.com/MLH-Fellowship/0.4.1-Quarantime/pull/15 is landed, you can easily integrate your component in that using copy-paste.

File: newsfeed/src/App.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import 'antd/dist/antd.css';
+import { Layout } from 'antd';
+
+import './App.css';
+import DetailsForm from './components/DetailsForm';
+import Project from './components/Project';
+
+class App extends React.Component {
+	constructor(props) {
```
Comment (saphal1998) [inline newsfeed/src/App.js:10]: @kunal-kushwaha @anikethsaha  For this, you can look into the componentDidMount() method, and we can just use setState to get the data from the Database

**Comment by kunal-kushwaha** [comment]
@saphal1998: Thank You! this really helped the team in designing the frontend. Closing the issue now

**Pull request closed without merging**